tests/integration/suite/placement/quorum/insecure.go

```diff
@@ -66,9 +67,8 @@ func (i *insecure) Setup(t *testing.T) []framework.Option {
 		placement.New(t, append(opts, placement.WithID("p3"))...),
 	}
 
-	fp.Free(t)
```

Comment:
We no longer need this?

Reply:
No, as it is now a proper process, this will happen during `Run`.